resources/views/importStatus.blade.php

```diff
@@ -1,9 +1,10 @@
 <x-layout>
     <div role="group" class="button-group right">
-        <a class="button primary progressive" href="{{ route('store.imports-overview') }}"><img src="{{ asset('/images/download-stats.svg') }}" class="button-icon" />{{__('store-layout.button:download-stats')}}</a>
+        <a class="button primary progressive" href="{{ route('store.imports-overview') }}"><img src="{{ asset('/images/download-stats-white.svg') }}" class="button-icon button-white-fill" />{{__('store-layout.button:download-stats')}}</a>
```

Comment:
Suggestion:  That extra class is probably not needed and can be dropped.

As I understand it, you can only affect SVGs with external CSS if they are literally in the DOM as HTML, not when they are used as an image in an `<img>` tag.

Reply:
hi, yes, i mentioned that i am still working on this. ;)